HTML:CSS learning project/index.html

```diff
@@ -23,69 +25,43 @@
-      Quasi libero illo quaerat autem facere incidunt suscipit velit qui, voluptates reprehenderit cumque nulla minima veniam repellat. Totam veritatis possimus alias. Odio fugit illum deserunt vero beatae quaerat distinctio iure.
-      Unde distinctio doloremque quia odit nisi nihil autem corrupti repudiandae fugit voluptate, inventore minus. Dolor rerum voluptas aspernatur quibusdam ab eos voluptatem distinctio sequi! Cupiditate dolorem adipisci deserunt porro repellat.
-      Nulla est optio rerum odit dolorum rem commodi provident e? Repudiandae ex reiciendis excepturi hic iure laboriosam numquam, sit maiores officia ipsa, esse adipisci expedita, dignissimos dolorem? Quam, eaque eveniet?
-      Nesciunt recusandae, quos voluptates animi odit temporibus deserunt magni, id error voluptate consectetur necessitatibus corrupti possimus quia omnis ipsum quis atque vel, facilis pariatur vitae nam repellat. Corrupti, ducimus eaque!
-      Minima doloremque enim voluptatem voluptates, quisquam nam fugit eos debitis neque optio dolores sunt nemo modi nulla mollitia, nostrum aspernatur quod velit officiis tempora ea et maxime. Eum, incidunt quidem.
-      Perferendis ut in nisi non libero asperiores quam itaque, eaque officia suscipit mollitia tempore enim sit explicabo delectus voluptates voluptatem, error, voluptate iure inventore ex optio quo cumque. Animi, cupiditate.
-      Laborum ducimus eius voluptate, porro id tempore. Sunt vitae quia excepturi qui reiciendis deleniti accusamus sapiente doloremque voluptatum error eos eveniet, nisi ratione blanditiis similique atque ullam suscipit a alias!
-      Neque asperiores tenetur voluptas sed! Laudantium enim voluptates autem veritatis mollitia corrupti itaque quos veniam consectetur voluptatibus id dolorum ut at aliquid, omnis unde sint et aperiam magni vitae ipsam?
+      <div class="row">
+        <div class="col-md">
+          <img src="images/image1.jpg">
+          <h2>Lorem ipsum dolor sit amet.</h2>
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. A, accusamus.</p>
+        </div>
+        <div class="col-md">
+          <img src="images/image2.jpg">
+          <h2>Lorem ipsum dolor sit amet.</h2>
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. A, accusamus.</p>
+        </div>
+        <div class="col-md">
+          <img src="images/image3.jpg">
+          <h2>Lorem ipsum dolor sit amet.</h2>
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. A, accusamus.</p>
+        </div>
+      </div>
     </div>
   </main>
   <footer class="main-footer">
-    <nav>
-      <a>contact us</a>
-    </nav>
-</footer>
+      <p class="footer-right"><a href="#">Back to top</a></p>
+      <p class="footer-left"> &copy; 2020 Road to web developer &middot;</p>
+      <a class="footer-left" href="">Home</a>
```

Comment:
Two classes with "footer-left" here. I'd recommend implement div element and wrap elements inside which should be in left side.